wgpu-core/src/device/global.rs

```diff
@@ -2083,7 +2083,7 @@ impl Global {
         self.hub.devices.remove(device_id);
     }
 
-    /// This closure will be called exactly once during "lose the device".
+    /// The closure might never be called.
```

Comment:
style: It's nice to use the same terms as the rest of the signature. In this case:

```suggestion
    /// `closure` might never be called.
```

This feedback also applies to the other doc. comments added in the `[wgpu-core] document which closures are guaranteed to be called ` commit.

I'd be open to simply adding this as a new commit, maybe even in a separate PR.